migrations/2017_12_10_094227_create_scores_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateScoresTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('scores', function (Blueprint $table) {
+            $table->increments('id');
+            $table->integer('team_id')->unsigned();
+            $table->integer('money')->unsigned(); // E.g. $10.34 would be stored as 1034
+            $table->integer('satisfaction')->unsigned(); // E.g. 98% would be stored as 98.
```

Comment:
I've seen this in production several times, but I've never been convinced about the actual need for it. On most implementations, a float has four byte precision, a double eight bytes. That's more than sufficient for anything we may want to do with the data, and it matches the type of what we do in our game as well. (e.g. money in-game is a float, not an int).

Just use a float for this, as it seems more natural.

Also, why would money be `unsigned'? :)

Reply:
I read online that for example 1-0.1 = 0.89999... . That's why I thought it's better to store money in cents. But if you say that's not necessary, I would gladly change it.

Reply:
And what about the satisfaction?

Reply:
`Also, why would money be `unsigned'?  `
We can't have a negative amount right?

Reply:
I believe we can, as in, I think I have come across cases where I ended up with a negative amount of money. Not sure if that's still the case, but I don't see any harm in allowing it into the database.

Satisfaction is a rate bounded between 0 and 100 (that is, 0 to 1 presented as a percentage). So just store it as a float, and that should do.

>1-0.1 = 0.89999

This is mostly correct, and part of a numeric rounding operation. However, the actual rounding is so small (on the order of machine precision) it is not a problem here at all. Furthermore, the number we present in the game suffers from the same issue: why do we accept it there, but wouldn't we in the database? I'd use a float: in our context, numeric rounding is not an issue.